.projenrc.js

```diff
@@ -193,6 +193,7 @@ const pythonCompatTask = project.addTask('integ:python-compat', {
 });
 const integTask = project.addTask('integ');
 integTask.spawn(project.buildTask);
+integTask.spawn(project.tasks.tryFind('package:python'));
 integTask.spawn(pythonCompatTask);
```

Comment:
This check should be enforced
```suggestion
integTask.spawn(project.tasks.tryFind('package:python'));
integTask.spawn(pythonCompatTask);
project.tryFindObjectFile('.mergify.yml').addOverride('pull_request_rules.0.conditions.3', 'status-success=integ');
```